formulas/mongo-express

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source "${BASH_SOURCE%/*}/../common"
+
+# Configs
+me_dock_name=dock-mongo-express
+me_dock_port=8081
+
+# Stop Container
+force_stop $me_dock_name
+
+# run Container
+run --detach \
+    --publish $me_dock_port:$me_dock_port \
+    --link dock-mongodb:mongo \
+    --name $me_dock_name \
+    knickers/mongo-express
+
+# Open in browser (OS X)
+container_ip=$(get_ip)
+open http://$container_ip:$me_dock_port
+echo "Now, mongo-express is show in browser."
```

Comment:
Please remove the last three lines. It is not dock's responsibility to open URLs in the browser.